packages/server/src/utils/pagination.ts

```
@@ -0,0 +1,20 @@
+import { z } from "@hono/zod-openapi";
+
+export const PaginationSchema = z.object({
+	page: z
+		.string()
+		.refine((value) => !isNaN(Number(value)), {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)